core/stream/datatype.go

```diff
@@ -108,7 +108,10 @@ func (t DataType) Format(f fmt.State, r rune) {
 }
 
 // Bits returns the size of the data type in bits.
-func (t *DataType) Bits() uint32 {
+func (t *DataType) Bits() uint32 {// F10 represents a 10-bit unsigned floating-point number.
+	F10 = DataType{Signed: false, Kind: &DataType_Float{&Float{5, 5}}}
```

Comment:
What's this doing here?

Reply:
That is a terrific question, I assume a copy-paste somewhere, will remove.